### PR TITLE
Fixed the login problem

### PR DESCRIPTION
**Pull request by adniyaYousaf**

### CODE REVIEW

File: client/src/pages/LogInForm.js [hunk 1]
```diff
@@ -30,9 +30,21 @@ function LogInForm() {
 			});
 			const data = await response.json();
 			if (response.ok) {
-				const token = data.resources.token;
+				const { token, user_type } = data.resources;
```
Comment (soonick) [inline client/src/pages/LogInForm.js]:
```suggestion
				const { token, user_type } = data.resource;
```

File: client/src/pages/LogInForm.js [hunk 2]
```diff
@@ -30,9 +30,21 @@ function LogInForm() {
...
+
 				localStorage.setItem("authToken", token);
-				navigate("/admin-dashboard");
+				localStorage.setItem("user_type", user_type);
```
Comment (soonick) [inline client/src/pages/LogInForm.js]: Just to be consistent, let's use camelCase, here too

```suggestion
				localStorage.setItem("userType", user_type);
```

File: client/src/pages/LogInForm.js [hunk 3]
```diff
@@ -30,9 +30,21 @@ function LogInForm() {
...
+				switch (user_type) {
+					case "admin":
+						navigate("/admin-dashboard");
+						break;
+					case "buyer":
+						navigate("/buyer-dashboard");
+						break;
+					default:
+						navigate("/bidder-dashboard");
+				}
```
Comment (soonick) [inline client/src/pages/LogInForm.js:47]: Thanks for adding this

**Review by soonick — CHANGES_REQUESTED** [review]

File: client/src/pages/LogInForm.js (hunk 3 above)
Comment (adniyaYousaf) [reply client/src/pages/LogInForm.js:47]: We are using different endpoints for each like for buyer `/publish-tender` . Shall I update them?

**Review by soonick — APPROVED** [review]
Thank you for doing this so quickly

**Pull request merged**